src/views/layouts/error.hbs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      {{>head}}
+      <title> {{title}} </title>
+      <link rel="stylesheet" href="./css/{{css}}.css">
+   </head>
+   <body>
+      {{{body}}
+      <script src="./js/{{js}}.js"></script>
```

Comment:
I think you will pass js files as an array?

Reply:
same as I said above?